spec/negotiation.html

```diff
@@ -325,5 +325,35 @@ <h1>GetNumberOption ( _options_, _property_, _minimum_, _maximum_, _fallback_ )<
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-partitionpattern" aoid="PartitionPattern">
+      <h1>PartitionPattern ( _pattern_ )</h1>
+
+      <p>
+        The PartitionPattern abstract operation is called with argument _pattern_. The following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _result_ be a new empty List.
+        1. Let _beginIndex_ be Call(%StringProto_indexOf%, _pattern_, &laquo; `"{"`, 0 &raquo;).
```

Comment:
This can now use the new intrinsics notation.
```suggestion
        1. Let _beginIndex_ be Call(%String.prototype.indexOf%, _pattern_, &laquo; `"{"`, 0 &raquo;).
```